Activities/Database/UiPath.Database/DatabaseConnection.cs

```diff
@@ -551,5 +561,41 @@ private static ArgumentDirection WokflowParameterDirectionToDbParameter(Paramete
                     throw new ArgumentException(Resources.ParameterDirectionArgumentException);
             }
         }
+
+        private bool UpdateDbParamType(DbParameter dbParameter, ParameterInfo parameterInfo)
```

Comment:
where is the return value of this method used?

Reply:
is not used at the moment..I will remove it
the idea is that in the future we can add the return value back
when changing the type of the parameter.. we want to change the most specific one then fallback to other 
some of the connections don't support changing the type of the param (eg. I tested with ODBC and Excel source)
Since there is a risk of breaking existing behavior I would change only what is strictly necessary (meaning for Oracle connection)